### PR TITLE
add title to microbit platform page

### LINKED ISSUE #94: Platform: Microbit doesn't have header
The Arduino platform page does, microbit does not. Just needs header text.

### PR DESCRIPTION
**Pull request by marksherman**
fixes #94

### CODE REVIEW

File: docs/src/app/pages/MicroBitIntro.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-BBC micro:bit is a computing platform from the BBC. It is an open platform for developing all manner of projects and is programmable by a number of different editors, including a [blocks editor](https://pxt.microbit.org) provided by Microsoft. Learn more about the micro:bit at the <a rel="nofollow" href="http://microbit.org/about/">Micro:bit Educational Foundation's website</a>.
+# Platform: BBC micro:bit
+
+The micro:bit is a computing platform from the BBC. It is an open platform for developing all manner of projects and is programmable by a number of different editors, including a [blocks editor](https://pxt.microbit.org) provided by Microsoft. Learn more about the micro:bit at the <a rel="nofollow" href="http://microbit.org/about/">Micro:bit Educational Foundation's website</a>.
```
Comment (ewpatton) [inline docs/src/app/pages/MicroBitIntro.md]: Do we want to make this <a> tag a target="_blank"?

**Review by ewpatton — COMMENTED** [review]
See inline comment

File: docs/src/app/pages/MicroBitIntro.md (hunk 1 above)
Comment (marksherman) [reply docs/src/app/pages/MicroBitIntro.md]: I didn't put that in, but I think its a good idea now that you've found it.

**Review by ewpatton — APPROVED** [review]
LGTM

**Pull request merged**